examples/riverpod_navigator2/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 
 
 dependencies:
+  auto_route: ^3.2.2
```

Comment:
The example shouldn't be using a third party package to work. Otherwise that's no-longer a Navigator 2 example but an autoRoute example

Reply:
What about [this one](https://github.com/PavelPZ/riverpod_navigator/tree/main/examples/riverpod_navigator_idea)?